internal/cloud/realm/access_list.go

```diff
@@ -81,3 +82,24 @@ func (c *client) AllowedIPCreate(groupID, appID, ipAddress, comment string, useC
 	}
 	return allowedIP, nil
 }
+
+func (c *client) AllowedIPUpdate(groupID, appID, allowedIPID, newAddress, comment string) error {
+	res, err := c.doJSON(
+		http.MethodPut,
```

Comment:
[question/nit] does this maybe want to be a `PATCH`?  from its usage, it looks like we'll only conditionally update the IP Address with values that are non-blank

Reply:
I think we can go with `PATCH` for this one, since we're updating an existing resource and not replacing it completely.

Reply:
id just check/run it by the rest of the project team first... if so, then its probably best left as a `TODO(REALMC-XXXX)` here so the corresponding backend change can go with it